app/src/main/java/com/sdp/movemeet/view/activity/ActivityDescriptionActivity.java

```diff
@@ -176,10 +181,8 @@ private void setButton() {
         }
     }
 
-    private void getParticipantNames() {
+    private void getParticipantNames(Activity activity) {
```

Comment:
Why do you pass the activity in argument ?

Reply:
Yes, good question! This is a needed change in order to be able to use this method of the `ActivityDescriptionActivity` for its corresponding test class. More precisely, this is to allow the "good flow" of the required parameters (contained in `activity`) through the tested sequence of functions.